Luke_Smith_Portfolio_Portfolio.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width initial-scale=1">
+	<title>Luke Smith Portfolio Portfolio</title>
+    <link rel="stylesheet" href="Luke_Smith_Portfolio.css">
+</head>
+<body>
+	<div style="background-color:white;">
```

Comment:
same as before, inline styles are bad. Give this div a class and apply a styling rule in your .css file